lekce2/14.py

```diff
@@ -0,0 +1,10 @@
+import sys
+word = sys.argv[1]
+text = word.split('_')
+capital1 = [cap.capitalize() 
+for cap in text]
+capital2 = [ str(cap[0].upper()) + cap[1:]  for cap in text]
```

Comment:
Aha, takhle! To me nanapadlo, jsem se tam furt pokousel nacpat ten `capitalize` nebo `title` :D

Reply:
:DDD jj :DD